### DISCUSSION TITLE
Copyright license

### DISCUSSION THREAD

**Discussion by GS-ST** [Okmeque1/software#2] [General]
DO u giv me permition to copy your error.md?

**Comment by Okmeque1** [comment]
I'll give permission to you if you 
1 : Use proper spelling. Spelling is important on GitHub as it shows that you are very professional
2 : Change the codes. If for example the FileNotFoundError is 6510B then it cannot be anything that has 6510 in it. Same thing for the other ones.

-Okmeque1

**Comment by Titan1umF0rge** [comment]
ok

**Comment by Titan1umF0rge** [comment]
completed
